midterm-2023-11-06/code/5215-berat-useini/script.ts

```diff
@@ -0,0 +1,191 @@
+type RockBand = {
+    ID: number;
+    name: string;
+    formed: number;
+    location: string;
+    country: string;
+    members: string;
+    albums: number;
+};
+
+interface albums{ 
+name: string;
+
+}
+
+document.addEventListener("DOMContentLoaded", siteCode);
+let rockband : RockBand[] = [];
```

Comment:
Since this is a collection of bands, that should be reflected in the name of the variable, i.e. `rockBands` vs `rockband`